mkdocs.yml

```diff
@@ -5,58 +5,89 @@ repo_url: https://github.com/ezsystems/user-documentation
 site_url: https://doc.ibexa.co/projects/userguide/en/latest/
 copyright: "Copyright 1999-2022 Ibexa AS and others"
 nav:
-    - 'Ibexa User Documentation': 'index.md'
-    - 'Getting started': 'getting_started.md'
-    - 'Content model': 'content_model.md'
-    - 'Creating content':
-        - 'Creating content': 'creating_content_basic.md'
-        - 'Translating content': 'translating_content.md'
-        - 'Creating forms': 'creating_forms.md'
-        - 'Editing images': 'editing_images.md'
-    - Tags: 
-        - 'Tags': 'taxonomy.md'
-    - Publishing:
-        - 'Publishing content': 'publishing/publishing.md'
-        - 'Editorial workflow': 'publishing/editorial_workflow.md'
-        - 'Scheduling publication': 'publishing/advanced_publishing_options.md'
-    - Organizing:
-        - 'Organizing the site': 'site_organization/organizing_the_site.md'
-        - 'Site Factory': 'site_organization/site_factory.md'
-        - 'Organizing the content': 'site_organization/organizing_the_content.md'
-        - 'Managing content in bulk': 'site_organization/multi_file_content_management.md'
-        - 'Pages': 'site_organization/working_with_page.md'
-    - 'Searching': 'search.md'
-    - 'User management': 'users/user_management.md'
-    - Shop administration:
-        - 'Shop administration': 'shop_administration/shop_administration.md'
-        - 'Customer management':
-            - 'Customer management': 'shop_administration/manage_users.md'
-            - 'Customer Portal': 'shop_administration/customer_portal.md'
-        - 'Dashboard': 'shop_administration/shop_dashboard.md'
-        - 'Orders': 'shop_administration/manage_orders.md'
-        - 'Navigation structure': 'shop_administration/navigation_structure.md'
-    - PIM:
-        - 'Products': 'shop_administration/manage_products.md'
-        - 'Product variants and assets': 'shop_administration/product_variants_and_assets.md'
-        - 'Product categories': 'shop_administration/product_categories.md'
-        - 'Prices and stock': 'shop_administration/manage_prices_and_stock.md'
-        - 'Catalogs': 'shop_administration/catalogs.md'
+    - Ibexa User Documentation: index.md
+    - Getting started:
+       - Get started: getting_started/get_started.md
+       - User interface: getting_started/discover_ui.md
+    - Content model management:
+        - Content model: content_management/content_model.md
+        - Work with Content Types: content_management/work_with_content_types.md
```

Comment:
That's because the content is sort of a melting pot of ideas, too. This will change once we have an actual "Create Content Type" article. Then this article will change to "Manage Content Type settings"